spyder_remote_services/__init__.py

```diff
@@ -1,7 +1,7 @@
 from spyder_remote_services.app import SpyderRemoteServices
 
 
-__version__ = '1.0.0'
+__version__ = '1.1.0'
```

Comment:
There hasn't been a public release of `1.0.0`, so we perhaps we should leave the same version?